src/Features/Core/Portable/AddImport/SearchScopes/MetadataSymbolsSearchScope.cs

```diff
@@ -50,15 +50,12 @@ public override SymbolReference CreateReference<T>(SymbolResult<T> searchResult)
                 SymbolFilter filter, SearchQuery searchQuery)
             {
                 var service = _solution.Services.GetService<SymbolTreeInfoCacheService>();
-                var info = await service.TryGetMetadataSymbolTreeInfoAsync(_solution, _metadataReference, CancellationToken).ConfigureAwait(false);
+                var info = await service.TryGetPotentiallyStaleMetadataSymbolTreeInfoAsync(_solution, _metadataReference, CancellationToken).ConfigureAwait(false);
```

Comment:
renamed to make the staleness possibility obvious.

Reply:
Is `PotentiallyStale` more clear than `Cached`? Caches always can be stale :)

Reply:
i would be ok with that :)